modules/distr-tracing-tempo-install-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-operator_{context}"]
+= Installing the {TempoName} Operator
+
+To install {TempoName}, you use the link:https://operatorhub.io/[OperatorHub] to install the {TempoName} Operator.
+
+By default, the Operator is installed in the `openshift-operators` project.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You have access to the cluster as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+* If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {TempoName} Operator.
```

Comment:
No required anymore. Replaced with object storage. [Same as Loki](https://docs.openshift.com/container-platform/4.10/networking/network_observability/installing-operators.html#network-observability-loki-installation_network_observability) but we don't support Swift.

Reply:
In that case, I'm removing it. Thank you.
```suggestion
```